src/libraries/Microsoft.Extensions.Primitives/src/Microsoft.Extensions.Primitives.csproj

```diff
@@ -10,17 +11,8 @@
   <ItemGroup>
     <Compile Include="$(CommonPath)Extensions\HashCodeCombiner\HashCodeCombiner.cs"
              Link="Common\src\Extensions\HashCodeCombiner\HashCodeCombiner.cs" />
-    <Compile Include="CancellationChangeToken.cs" />
-    <Compile Include="ChangeToken.cs" />
-    <Compile Include="CompositeChangeToken.cs" />
-    <Compile Include="Extensions.cs" />
-    <Compile Include="IChangeToken.cs" />
-    <Compile Include="InplaceStringBuilder.cs" />
-    <Compile Include="StringSegment.cs" />
-    <Compile Include="StringSegmentComparer.cs" />
-    <Compile Include="StringTokenizer.cs" />
-    <Compile Include="StringValues.cs" />
-    <Compile Include="ThrowHelper.cs" />
+    <Compile Include="$(CommonPath)System\Text\ValueStringBuilder.cs"
```

Comment:
It looks as though this is only needed to be compiled in when 

```
#if NETCOREAPP || NETSTANDARD2_1
```

is false. So we can add a condition here:

```
Condition="'$(TargetFramework)' == 'netstandard2.0' or
                        $(TargetFramework.StartsWith('net4'))"
```

or similar.